pii-filtering/index.mdx

```diff
@@ -0,0 +1,73 @@
+---
+title: 'PII Filtering'
+backlink_href: /docs
+backlink_title: 'Documentation'
+---
+
+[Personally Identifiable
+Information](https://csrc.nist.gov/glossary/term/personally_identifiable_information)
+(PII) about your users must be handled very carefully in order to respect your
+users' privacy and to comply with laws and regulations your organization is
+subject to. When a third-party tool like pganalyze processes your database
+statistics and logs, it may involve handling data that includes this PII.
+
+In order to strike the right balance between what you share with pganalyze and
+the impact on monitoring, statistics, and insights available through our
+service, we have several collector settings that control metadata collection.
+
+Note that this filtering happens in the collector, the open-source agent that
+connects to your database and runs on your infrastructure. Filtering occurs
+before the data ever reaches pganalyze.
+
+
+## Filtering settings
+
+The `filter_log_secret` setting determines how content coming from logs is filtered:
+ - `credential`: Passwords and other credentials, like private keys
+ - `parsing_error`: Text logged by Postgres for parsing errors (could contain anything, including credentials)
+ - `statement_text`: All statement texts (which may contain table data if not using bind parameters)
+ - `statement_parameter`: Bind parameters for a statement (which may contain table data for INSERT statements)
```

Comment:
it's not only with INSERT statements, I believe (e.g. bind parameter of UPDATEs or DELETEs or even SELECTs can contain some data)

Reply:
I took that from [here](https://github.com/pganalyze/collector/blob/main/state/logs.go#L82), but yeah, we should be clearer here. Maybe
```suggestion
 - `statement_parameter`: Bind parameters for a statement (which may contain table data for statements like INSERT or UPDATE)
```
? Technically you can also get table data in SELECTs and DELETEs, but most of the time, those will just pass an id as a parameter. I think some user discretion is necessary in determining how their particular database fits into this, so we don't need to be exhaustive here, we just need to make it clear the list is not complete.

Reply:
I like the suggested version 👍 agreed that we don't need to be exhaustive here, but former one could wrongly imply that it's only for INSERTs so I think it's nice to update like this